### DISCUSSION TITLE
Can not retrieve data

### DISCUSSION THREAD

**Discussion by muhamedRadwan** [spatie/laravel-settings#73] [Q&A]
When I  tring to get data from My Model I get Error in App service Provider
here is my code
![image](https://user-images.githubusercontent.com/16479089/122663657-97fd5f00-d19c-11eb-948b-d8464da366fd.png)
here is the error
![image](https://user-images.githubusercontent.com/16479089/122663664-a0ee3080-d19c-11eb-9deb-d3d6486d06b4.png)
my config
![image](https://user-images.githubusercontent.com/16479089/122663684-bb280e80-d19c-11eb-85d4-52980ee959ad.png)
My Model 
![image](https://user-images.githubusercontent.com/16479089/122663702-cf6c0b80-d19c-11eb-9f28-bc3c5df35a39.png)
DB Table
![image](https://user-images.githubusercontent.com/16479089/122663732-eb6fad00-d19c-11eb-997f-63e58ddff956.png)

Any help?

**Comment by ragingdave** [comment]
I think you are missing the properties that actually define the settings you are trying to access. So for favicon, you need a property on your Setting class like `public string $favicon`, then it should work, but you have to add all those as properties on the class that you noted from your phpmyadmin screenshot.

**Reply by muhamedRadwan** [reply]
Hi, ragingdave
when I add properties to setting Class I don't reserve error but it value always null 
but This worked for me 
(object)(new DatabaseSettingsRepository(config('settings.repositories.database')))->getPropertiesInGroup('general');
But I think this is wrong way
